### PR TITLE
chore: add tab index to powered by algolia text

### PR DESCRIPTION
**Pull request by harish-sethuraman**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

#### What changes did you make? (Give an overview)
copy of https://github.com/eslint/eslint.org/pull/615

### CODE REVIEW

**Review by amareshsm — APPROVED** [review]
LGTM. waiting for 2nd approval.

**Review by fasttime — APPROVED** [review]
LGTM. Thanks!

**Comment by nzakas** [comment]
This isn't the right way to solve the original problem. We shouldn't be removing links from the tab order. I'm reverting this and we can think of a different solution.

**Pull request merged**
